bindings/alicloud/rocketmq/rocketmq.go

```diff
@@ -90,11 +90,11 @@ func (a *AliCloudRocketMQ) Read(ctx context.Context, handler bindings.Handler) e
 		if topicStr == "" {
 			continue
 		}
-		mqType, mqExpression, topic, err := parseTopic(topicStr)
+		mqType, mqExpression, topic, err1 := parseTopic(topicStr)
 		if err != nil {
```

Comment:
**if** you need to use `err1` (not sure you do, but **if**), then this needs to change to `err1` too

Reply:
See my other suggestion related to this.